sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BulkItemRequestOptionsBase.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.RequestOptions;
+import com.azure.cosmos.util.Beta;
+
+/**
+ * Encapsulates options that can be specified for an operation used in Bulk execution. It can be passed while
+ * creating bulk request using {@link BulkOperations}.
+ */
+@Beta(value = Beta.SinceVersion.V4_9_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public class BulkItemRequestOptionsBase {
+
+    protected String ifMatchETag;
+    protected String ifNoneMatchETag;
+    protected Boolean contentResponseOnWriteEnabled;
+
```

Comment:
package private ctor is missing to avoid that users can create their own

BulkItemRequestOptionsBase() {
}

Reply:
private ctor won't work. I've made protected constructor.